data/fields/fence_type.json

```
@@ -1,5 +1,22 @@
 {
     "key": "fence_type",
     "type": "combo",
-    "label": "Type"
+    "label": "Type",
+    "strings": {
+        "options": {
+            "barbed_wire": "Barbed Wire",
+            "bars": "Vertical Bars",
+            "chain_link": "Chain-Link",
+            "electric": "Electric",
+            "pailing": "Pailing",
+            "panel": "Panels",
+            "pole": "Poles",
+            "post_and_rail": "Post-And-Rail",
+            "railing": "Railing",
+            "roundpole": "Roundpole",
+            "slatted": "Slatted Boards",
+            "split_rail": "Spit-Rail",
+            "wire": "Wire"
```

When I was looking into it, I was wondering how to deal with all the alternative terms. I think it would be better to list them in the translation than to not expose them at all. Otherwise the translations is kind of a one-to-one version of the value.

```suggestion
            "barbed_wire": "Barbed Wire",
            "bars": "Vertical or metal Bars / Pool Fence",
            "chain_link": "Chain-link / Wire mesh / Chain-wire",
            "electric": "Electric",
            "pailing": "Pailing",
            "panel": "Panels",
            "pole": "Poles",
            "post_and_rail": "Post-And-Rail",
            "railing": "Railing",
            "roundpole": "Roundpole",
            "slatted": "Slatted Boards",
            "split_rail": "Spit-Rail / Log Fence",
            "wire": "Wire"
            "concrete": "Concrete blocks / Special geometry concrete",
            "metal": "Generic metal fence",
            "wood": "Generic wooden fence",        
```